### DISCUSSION TITLE
Map days played in the save file info

### DISCUSSION THREAD

**Discussion by Branikolog** [ihhub/fheroes2#6707] [Ideas]
It would be nice if the save file info would show a hint about the day it was created.
![Save file info](https://user-images.githubusercontent.com/55348946/221169736-603c8fda-86d1-4d0b-9e33-c58faf0fa7b8.png)

Just a simple string saying:
Month: 1, Week: 2, Day: 3.
This would help to navigate in save file list.

**Comment by oleg-derevenetz** [comment]
Implemented in #6714

**Comment by Branikolog** [comment]
Nice work, @oleg-derevenetz !
